recipe/meta.yaml

```diff
@@ -43,3 +44,5 @@ about:
 extra:
   recipe-maintainers:
     - lvoliveira
+    - nicoddemus
+    - igortg
```

Comment:
@igortg please confirm that you want to be a maintainer here.

Reply:
Ok, no problem